packages/client/src/components/book-copy-details-row.vue

```diff
@@ -43,19 +51,49 @@
 
         <template v-else-if="name === 'problems'">
           <problems-button
+            v-if="!isMobile"
             :book-copy="bookCopy"
             @update-problems="emit('updateProblems')"
           />
         </template>
 
         <template v-else-if="name === 'history'">
           <q-btn
+            v-if="!isMobile"
             :icon="mdiHistory"
             color="primary"
             flat
             round
             @click="emit('openHistory', bookCopy)"
           />
+
+          <q-btn
```

Comment:
Dunno if it's feasible with the whole action column, but for sure the btn + menu can be extracted into its own component applying it in one of the instances

Please design how you would do the extraction, present it to me and ONLY THEN proceed replacing it in all cases